src/components/labels/Labels.tsx

```diff
@@ -0,0 +1,41 @@
+import {useEffect, useState} from 'react';
+import {
+  getLabels,
+  LabelsResponse,
+  Label as ImageLabel,
+} from 'src/resources/TagResource';
+import {Label} from '@patternfly/react-core';
+
+export default function Labels(props: LabelsProps) {
+  const [labels, setLabels] = useState<ImageLabel[]>([]);
+  useEffect(() => {
+    (async () => {
+      try {
+        const labelsResp: LabelsResponse = await getLabels(
+          props.org,
+          props.repo,
+          props.digest,
+        );
+        setLabels(labelsResp.labels);
+      } catch (error: any) {
+        console.log('Unable to get labels: ', error);
+      }
+    })();
+  }, []);
+
+  return (
+    <>
+      {labels.map((label: ImageLabel) => (
+        <Label key={label.key}>
+          {label.key} = {label.value}
+        </Label>
+      ))}
+    </>
+  );
+}
+
+type LabelsProps = {
```

Comment:
we have both `type` and `interface` in code. We should decide on one and stick to it. Let's discuss this

Reply:
Yeah I agree. It seems to be split 50/50. We can discuss this in a separate thread and then make the changes in a separate PR?

Reply:
Changed the references in this PR to interfaces. Another PR will be made to fix the rest.